imap/append.c

```diff
@@ -331,8 +333,8 @@ EXPORTED FILE *append_newstage_full(const char *mailboxname, time_t internaldate
     stage = xmalloc(sizeof(struct stagemsg));
     strarray_init(&stage->parts);
 
-    snprintf(stage->fname, sizeof(stage->fname), "%d-%d-%d",
-             (int) getpid(), (int) internaldate, msgnum);
+    snprintf(stage->fname, sizeof(stage->fname), "%d-%llu",
+             (int) getpid(), (long long unsigned)append_counter++);
```

Comment:
Add `#include <inttypes.h>`, and then use the proper uint64_t formatter, instead of the longwinded cast

```suggestion
    snprintf(stage->fname, sizeof(stage->fname), "%d-%" PRIu64,
             (int) getpid(), append_counter++);
```

(Check this before integrating! I'm typing in a github window, not my editor...)

Reply:
Fixed, ta